### PR TITLE
Fixes #12758: Fix seeds test to not need DatabaseCleaner

### PR DESCRIPTION
**Pull request by ehelms**
This fixes failures that occur on Postgres 8.4 from primary key
failures.

### CODE REVIEW

**Comment by ehelms** [comment]
@parthaa @daviddavis could you please review this? My testing on an EL6 devel box indicated this would fix the issue. Currently, I have test_katello_core locked to EL7 boxes only so I can relax that in hopes this hits as an EL6 box, or you can test it out yourselves.

File: test/lib/tasks/seeds_test.rb [hunk 1]
```diff
@@ -1,90 +1,94 @@
...
-    DatabaseCleaner.clean_with :truncation
-    Setting.stubs(:[]).with(:administrator).returns("root@localhost")
-    Setting.stubs(:[]).with(:send_welcome_email).returns(false)
-    Katello::Repository.stubs(:ensure_sync_notification)
-  end
+module Katello
+  class SeedsTest < ActiveSupport::TestCase
+    setup do
+      # Disable AR transactional fixtures as we use DatabaseCleaner's truncation
+      # to empty the DB of fixtures for testing the seed script
```
Comment (daviddavis) [inline test/lib/tasks/seeds_test.rb]: I think we can remove these comments?

**Comment by daviddavis** [comment]
ACK from me

**Comment by parthaa** [comment]
apj

**Comment by ehelms** [comment]
Comments removed

**Pull request merged**